include/opcode/riscv-opc.h

```diff
@@ -2787,6 +2787,40 @@
 #define CSR_VTYPE 0xc21
 #define CSR_VLENB 0xc22
 #endif /* RISCV_ENCODING_H */
+
+/* CORE-V Specific Instructions  */
+/* Multiply accumulate */
+#define MATCH_CV_MAC 0x42000033
+#define MATCH_CV_MSU 0x42001033
+
+#define MASK_CV_MACMSU 0xFE00707F
+
+#define MATCH_CV_MULS      0x8000005B
+#define MATCH_CV_MULHHS    0xC000005B
+#define MATCH_CV_MULU      0x0000005B
+#define MATCH_CV_MULHHU    0x4000005B
+
+#define MASK_CV_MULSH	0xFE00707F
+
+#define MATCH_CV_MULSN     0x8000005B
+#define MATCH_CV_MULHHSN   0xC000005B
+#define MATCH_CV_MULSRN    0x8000405B
```

Comment:
Is it okay that some values are the same?
We could tidy up by merging MATCH_CV_MULS and MATCH_CV_MULSN together, etc.

Reply:
I am happy to merge them if we agree it is a good idea :)

Reply:
The duplicate values got removed when I cherry-picked the new encodings, thus this has been resolved (check [here](https://github.com/pietraferreira/binutils-gdb/pull/1/commits/2a0274cb2abaaa7f4cb5c730a10379438e4ef423).